src/index.ts

```diff
@@ -55,21 +55,43 @@ const argv = yargs
     default: 'INFO',
   }).argv
 
+const {
+  'ethereum-node-url': ethereumNodeUrl,
+  'page-size': pageSize,
+  port,
+  'max-hops': maxHops,
+  'poll-frequency': pollFrequency,
+  'price-rounding-buffer': priceRoundingBuffer,
+  'num-threads': numThreads,
+  'base-path': basePath,
+  verbosity,
+} = argv
```

Comment:
I think `yargv` already does that. Try
```suggestion
const {
  ethereumNodeUrl,
  pageSize,
  port,
  maxHops,
  pollFrequency,
  priceRoundingBuffer,
  numThreads,
  basePath,
  verbosity,
} = argv
```

Reply:
Interesting, didn't know yargs also parse it as camel case. 
The issue I'm having is that their typescript don't recognise the camelCase version type

![image](https://user-images.githubusercontent.com/2352112/80473042-dbb45a00-8945-11ea-97cc-7a5b672fed67.png)

![image](https://user-images.githubusercontent.com/2352112/80473133-00103680-8946-11ea-9c82-98786911c64e.png)

I see ways around it, but the one I added seems the less verbose one of the ones I can think of.